docs/tutorial.md

```diff
@@ -221,8 +220,11 @@ surrounding the current work item.
 
 Lastly, there are two more things of note for the call to `parallel_for`: The
 first is the **kernel name**. Just like in SYCL, each kernel function in
-Celerity has to have a unique name in the form of a template type parameter.
+Celerity may have a unique name in the form of a template type parameter.
 Here we chose `MyEdgeDetectionKernel`, but this can be anything you like.
+
+> Kernel names used to be mandatory in SYCL 1.2.1 but have since become optional.
```

Comment:
For ComputeCpp, they are still required.

Reply:
I mean, the statement is still true as it stands. I'd rather keep it concise here, given that the user will be presented with a nice static assertion telling them that they need a name for CCPP!

Reply:
True, technically that's not a statement on Celerity.